test/e2e/apps/job.go

```diff
@@ -49,9 +49,20 @@ var _ = SIGDescribe("Job", func() {
 		gomega.Expect(err).NotTo(gomega.HaveOccurred(), "failed to ensure job completion in namespace: %s", f.Namespace.Name)
 	})
 
+<<<<<<< HEAD
 	// Pods sometimes fail, but eventually succeed.
 	ginkgo.It("should run a job to completion when tasks sometimes fail and are locally restarted", func() {
 		ginkgo.By("Creating a job")
+=======
+	/*
+		Release : v1.15
+		Testname: Jobs, completion after task failure
+		Description: Create a job. Explicity cause the tasks fails once intially and then they should be restarted locally.
+						After the restart ensure that the job eventually successds and reaches completion.
```

Comment:
Please lint the lines appropriately. Multiline Conformance Doc would do great, as it is now but with proper lint.

Reply:
What about:
Description: Create a job. Explicitly cause the tasks to fail once initially and then they should be restarted locally.
After the restart, Job MUST eventually succeed and reach to completion.